perses/app/setup_relative_calculation.py

```diff
@@ -906,6 +938,13 @@ def run(yaml_filename=None):
                 _logger.info(f"\t\tFinished phase {phase}")
 
 def _resume_run(setup_options):
+    from openmmtools.cache import ContextCache
+    # get platform
+    platform = get_openmm_platform(platform_name=None)
```

Comment:
I will create the issue for the milestone for this, if we agree.

Reply:
Created the issue https://github.com/choderalab/perses/issues/939 

At this point it is defaulting to `None` and picking the best platform with mixed precision. But it will be just adding the CLI/YAML option and make `get_open_platform` use that once we have it.

It is not a big change, but it just doesn't feel like a "bugfix". If we think we should change this in this bugfix release, let me know. Thanks!